README.md

```diff
@@ -113,7 +113,7 @@ The provided query editor is a standard SQL query editor. Grafana includes some
 | `$__dateFilter(column)`              | `$__dateFilter` creates a conditional that filters the data (using `column`) based on the date range of the panel.                                                                                                                  | `$__date(my_date)`                                            | `my_date BETWEEN date '2017-07-18' AND date '2017-07-18'`                                             |
 | `$__parseTime(column,format)`        | `$__parseTime` cast a varchar as a timestamp with the given format.                                                                                                                                                                 | `$__parseTime(eventtime, 'yyyy-MM-dd''T''HH:mm:ss''Z')`       | `parse_datetime(time,'yyyy-MM-dd''T''HH:mm:ss''Z')`                                                   |
-| `$__timeFilter(column,format)`       | `$__timeFilter` creates a conditional that filters the data (using `column`) based on the time range of the panel. The second argument is used to optionally parse the column from a varchar to a timestamp with a specific format. | `$__timeFilter(time, 'yyyy-MM-dd HH:mm:ss')`                  | `TIMESTAMP time BETWEEN TIMESTAMP '2017-07-18T11:15:52Z' AND TIMESTAMP '2017-07-18T11:15:52Z'`        |
+|  `$__timeFilter(column,format)`       | `$__timeFilter` creates a conditional that filters the data (using `column`) based on the time range of the panel. The second argument is used to optionally parse the column from a varchar to a timestamp with a specific format. Keep in mind that this macro uses Presto's [Java Date Functions](https://prestodb.io/docs/current/functions/datetime.html#java-date-functions) `parse_datetime(string, format)` when a custom format is passed as `format` argument.| 1. Without specifying a format: `$__timeFilter(time)` <br /><br />2. Using the [default format](https://github.com/grafana/athena-datasource/blob/main/pkg/athena/macros.go#L14): `$__timeFilter(time, 'yyyy-MM-dd HH:mm:ss')`<br /><br />3. With another custom format: `$__timeFilter(time, 'yyyy-MM-dd''T''HH:mm:ss''+0000')` | 1. Without specifying a format: `time BETWEEN TIMESTAMP '2017-07-18 11:15:52' AND TIMESTAMP '2017-07-18 11:25:52'`<br /><br />2. Using the [default format](https://github.com/grafana/athena-datasource/blob/main/pkg/athena/macros.go#L14): `TIMESTAMP time BETWEEN TIMESTAMP '2017-07-18T11:15:52Z' AND TIMESTAMP '2017-07-18T11:15:52Z'`<br /><br />3. With another custom format: `parse_datetime(time,'yyyy-MM-dd''T''HH:mm:ss''+0000') BETWEEN TIMESTAMP '2017-07-18 11:15:52' AND TIMESTAMP '2017-07-18 11:25:52'`|
```

Comment:
for the link, can you use a version? so the link will always point to the code at a certain time. Like https://github.com/grafana/athena-datasource/tree/v1.0.3/pkg/athena/macros.go#L14

Reply:
Right!